index.html

```diff
@@ -95,6 +95,9 @@ <h1 class="headline-heading">
 <!-- Work section also falls under middle section -->
 <section id="portoliolarge-portfolio"></section>
 
+
+<section id="portfolio" class="works-section">
+  
```

Comment:
You're welcome @kayjeee However once you have completed the changes, you need to request a review on your dashboard so that it can be reviewed :wink: